nBTC/sources/utxo.move

```diff
@@ -27,28 +28,24 @@ const INACTIVE_BONUS: u64 = 200;
 const NO_CHANGE_BONUS: u64 = 1_000;
 const DUST_PENALTY: u64 = 200;
 
-// TODO: we need to store them by owner (the nBTC key)?
 public struct Utxo has copy, drop, store {
```

Comment:
```suggestion
public struct Utxo has copy, store {
```

Reply:
let' s remove `drop` to avoid mistakes. UTXO is an asset.
You can add `consumeUtxo` funciton to explicitly drop it.

Reply:
If you have a copy, then UTXO just copy and drop is okay here :D